.gitignore

```diff
@@ -36,34 +36,15 @@ test.key
 /editorconfig-checker
 /staticcheck
 
-ui/node_modules
```

Comment:
Did some cleanup in here not directly related to this PR since we no longer have these things in the `ui` directory.